### DISCUSSION TITLE
question: Where is the web cache storage persisted to disk?

### DISCUSSION THREAD

**Discussion by jsejcksn** [denoland/deno#25205] [Q&A] [answered]
Can anyone provide information about where the [cache](https://docs.deno.com/api/web/cache) storage is persisted to disk? I thought it was in origin storage (e.g. `$DENO_DIR/location_data`), but I don't see new entries being created after the creation of new cache entries in modules that resolve to different origin keys. Conversely, when I use the web storage or KV APIs in those same modules, I do see new entries in the origin storage directory…

/cc @bartlomieju because [you recommended](https://discord.com/channels/684898665143206084/684898665151594506/1277257721833324658) that I ask this here

**Comment by satyarohith** [comment, marked as answer]
Hi @jsejcksn. It's currently stored under "deno_cache" in the temp directory https://github.com/denoland/deno/blob/2bb013f9baa927fc7b392766b0182b91380b9aa8/cli/worker.rs#L764-L766

**Reply by jsejcksn** [reply]
Thanks for the response and source link!

**Comment by jsejcksn** [comment]
Ref:
- https://github.com/denoland/deno/pull/26205
